src/controller/python/chip/yaml/YamlUtils.py

```diff
@@ -0,0 +1,130 @@
+#
```

Comment:
could we avoid names like utils/helpers/misc ? They become a grab-all for unrelated functionality

Maybe `convert` is what this mostly does and we could separate out the exceptions/errors as their own thing

Reply:
Haha... It is like you see my locally (not yet uploaded) follow up commits that handle other elements of the yaml fields starting to make their way into this file because of that utils name :laughing: